docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: '3.4'
+
+x-common-config: &common_config
+  volumes:
+    - home:/home
+  environment:
+    - SQUAD_CELERY_BROKER_URL=amqp://rabbitmq
+    - XDG_DATA_HOME=/home
+
+volumes:
+  home:
+
+services:
+  rabbitmq:
+    image: rabbitmq
+
+  dbmigrate:
+    <<: *common_config
+    image: squad
```

Comment:
Is it mandatory to build image before running 'docker-compose up'? Without building the image I got the error: 
```
Pulling dbmigrate (squad:)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]y
Pulling dbmigrate (squad:)...
ERROR: pull access denied for squad, repository does not exist or may require 'docker login'
```
I built the image locally and tagged it 'squad'. This did the trick, but I didn't find it described anywhere. Maybe we should replace 'image: squad' with
```
build:
  context: ./
```
This will build image from the local sources.

Reply:
And my image turns out to be "bad":
```
django.template.library.InvalidTemplateLibrary: Invalid template library specified. ImportError raised when trying to load 'squad.ci.templatetags.lava': No module named 'ruamel'
```

Reply:
added the `build:` entry, and it works for me. docker-compose said:

```
WARNING: Image for service dbmigrate was built because it did not already exist.
To rebuild this image you must use `docker-compose build` or `docker-compose up --build`.
```